mem-01-b.xml

```diff
@@ -7083,7 +7173,7 @@ When used with {-lu':v}, this prefix means "Someone (indefinite subject) does so
       <column name="search_tags_de"></column>
       <column name="search_tags_fa"></column>
       <column name="search_tags_sv"></column>
-      <column name="search_tags_ru"></column>
+      <column name="search_tags_ru">деление</column>
```

Comment:
Redundant?

Reply:
Why the translation in search tags is redundant?For example look here https://github.com/barricade86/klingon-assistant-data/blob/master/mem-01-b.xml#L7139 ?Is that redundant too?

Reply:
It appears to be the same as the definition.

The tags are for additional terms not in the definition which someone might search for. For example, instead of "divide", someone might search for "division". But your search tag appears to be identical to your definition?

Reply:
I am not invite you, to debate, but why in this example english `search_tag` and `definition` differs from each other? `definition` is divide, and `search_tags` is division. I can make the value of `search_tags` and `definition` equals, but first i want to know the answer on my question

Reply:
The English definition is always as close to the canon definition given by Marc Okrand as possible. But sometimes the definition doesn't capture all possible words a person might search for, for which the Klingon word is an appropriate result. Even though **boqHa''egh** is defined as "divide (used in math)", a person searching for the mathematical operation of division might type "division" as their search term. You shouldn't make the `search_tags` and `definition` equal, as then the `search_tags` would be redundant. Does that make sense?